pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -1287,9 +1288,23 @@ func (m *kubeGenericRuntimeManager) removeContainerLog(ctx context.Context, cont
 	if status == nil {
 		return remote.ErrContainerStatusNil
 	}
+	// Remove file used for termination log
+	labeledInfo := getContainerInfoFromLabels(status.Labels)
+	containerPath, ok := status.Annotations[containerTerminationMessagePathLabel]
+	if ok {
+		subHostPath := filepath.Join(config.DefaultKubeletPodsDirName, string(labeledInfo.PodUID), config.DefaultKubeletContainersDirName, labeledInfo.ContainerName)
+		for _, mount := range status.Mounts {
+			if mount.ContainerPath == containerPath && strings.Contains(mount.HostPath, subHostPath) {
+				err := m.osInterface.Remove(mount.HostPath)
+				if err != nil && !os.IsNotExist(err) {
+					klog.ErrorS(err, "Failed to remove file used for termination log", "terminationMessagePath", mount.ContainerPath, "hostPath", mount.HostPath)
```

Comment:
If somehow we have a bug in this logic and get a large number of these messages, we may want to disable this change in kubelet. Should we introduce the feature gate for this change? (not a KEP, simply the feature gate)